File: backend/src/main/java/softeer/wantcar/cartalog/entity/option/Option.java
package softeer.wantcar.cartalog.entity.option;

import softeer.wantcar.cartalog.entity.HashTag;

import java.util.List;

public class Option {
    private Long id;
    private String name;
    private String type;
    private String category;
    private String description;
    private String imageUrl;
    private List<SubModelOption> subModelOptions;
    private List<TrimOption> trimOptions;
    private List<HashTag> hashTags;
}

File: Android/app/src/main/java/com/softeer/cartalog/data/model/Option.kt
package com.softeer.cartalog.data.model

data class Option(
    val id: String,
    val name: String,
    val parentCategory: String?,
    val childCategory: String?,
    val imageUrl: String,
    val price: Int,
    val chosen: Int,
    val hashTags: List<String>
)
